infrateam_first_responder.md

```diff
@@ -87,13 +87,9 @@ Then, **warn #dlss-infra-stage-qa-use** of the impending deployment to stage in
 
 Then **run infrastructure-integration-tests** (see [documentation](#run-infrastructure-integration-tests) below) after deploy to stage.
 
-#### 5. Deploy to QA
+#### 5. Deploy to prod
 
-Then, **warn #dlss-infra-stage-qa-use** of the impending deployment to QA in case there is active testing going on; if so, be sure to either comment out that app or coordinate with tester and then deploy the tag you created above to QA using `sdr-deploy`.
-
-#### 6. Deploy to prod
-
-Finally, **warn #dlss-infra-chg-mgmt** of the impending deployment to prod, and then deploy the tag you created above to prod using `sdr-deploy`.
+**Warn #dlss-infra-chg-mgmt** of the impending deployment to prod, and then deploy the tag you created above to prod using `sdr-deploy`.
 
 Next, **turn off Google Books when deploying to production** at https://sul-gbooks-prod.stanford.edu/features. This avoids failed deposit due to a temporary Cocina model mismatch. Unlike other applications, the deposits will fail without retry and require manual remediation.  Don't forget to re-enable when prod deployments are complete!
```

Comment:
This should happen *before* the deploy step in the previous paragraph. It was confusing before, may as well fix now.